spec/features/form/tasklist_page_spec.rb

```diff
@@ -26,19 +26,20 @@
     sign_in user
   end
 
-  it "skips to the first section if no answers are completed" do
+  it "shows if the section has not been started" do
     visit("/logs/#{empty_case_log.id}")
-    expect(page).to have_link("Skip to next incomplete section", href: /#household_characteristics/)
+    expect(page).to have_content("This log has not been started.")
   end
 
-  it "shows the number of completed sections if no sections are completed" do
+  it "shows number of completed sections if one section is completed" do
+    answer_all_questions_in_income_subsection(empty_case_log)
     visit("/logs/#{empty_case_log.id}")
-    expect(page).to have_content("You have completed 0 of 9 sections.")
+    expect(page).to have_content("1 of 9 sections completed.")
   end
 
-  it "shows the number of completed sections if one section is completed" do
+  it "show skip link for next incomplete section" do
     answer_all_questions_in_income_subsection(empty_case_log)
     visit("/logs/#{empty_case_log.id}")
-    expect(page).to have_content("You have completed 1 of 9 sections.")
+    expect(page).to have_link("Skip to next incomplete section", href: /#household-characteristics/)
```

Comment:
Would be great to add one more test here for what should happen when the link is clicked (what html change do we expect?).

Reply:
Is it possible to test that page has scrolled to the `:target`? Maybe test if href has `#household-characteristics` appended?

Reply:
Is that all that's supposed to happen now? No highlighting any more? In that case yes that's probably the best check.

Reply:
Okay, how might I write that test 😉

Reply:
Something like this:

```
context "when the 'Skip to next incomplete section is clicked'" do
    it "jumps to the next incomplete section" do
      answer_all_questions_in_income_subsection(empty_case_log)
      visit("/logs/#{empty_case_log.id}")
      click_link("Skip to next incomplete section")
      expect(page).to have_current_path("/logs/#{empty_case_log.id}#household-section")
    end
  end
```

Reply:
Doesn't look like the functionality actually works at the moment though